phpMain/arithmatic.php

```diff
@@ -45,3 +27,21 @@ function calculus ($a, $b) {
 }
 
 echo calculus (9, 3);
+
```

Comment:
Why are u creating the same function again and again? This way, you are overriding the behavior of your function. For e.g if I want to do addition I won't be able to do it because your division implementation overrides every other implementation.